5_databases/week1/Homework1

```diff
@@ -0,0 +1,47 @@
+-- Active: 1706445451493@@127.0.0.1@3306@HYFSql
+select * from status
+
+Find out how many tasks are in the task table
+
+select count(id) as task_count from task;
+Find out how many tasks in the task table do not have a valid due date
+
+select count(id) as no_due_date_count from task where due_date IS NULL OR due_date < NOW();
```

Comment:
This the correct query but the due_date < NOW() part is not required. these dates are still valid it's just the task is due. The requirement was to check for tasks with no due date only (due_date IS NULL) :)